BAZProject/Controllers/TrendingViewController.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  TrendingViewController.swift
+//  BAZProject
+//
+//
+
+import UIKit
+
+class TrendingViewController: UITableViewController {
+    
+    private var moviesListVM: MovieListViewModel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setup()
+    }
+
+}
+
+// MARK: - Functions
+
+extension TrendingViewController {
+    
+    /**
+     Configures the view controller by loading trending movies for the day and populating the table view with the results.
+    */
+    private func setup() {
+
+        let trendingMovieDayResource = Resource<MovieList>(url: Endpoint.trendingMovieDay.url) { data in
+            return try? JSONDecoder().decode(MovieList.self, from: data)
+        }
+        
+        MovieAPI().load(resource: trendingMovieDayResource) { (result) in
```

Comment:
Al trabajar con closures es importante como desarrolladores, mantener un uso correcto de la memoria. En este caso hace falta agregar un capture list para asegurarnos de no generar un memory leak.

```suggestion
        MovieAPI().load(resource: trendingMovieDayResource) { [weak self] result in
```